FluentTerminal.App/Views/MainPage.xaml.cs

```diff
@@ -95,5 +97,44 @@ private void UpdateLayoutMetrics()
                 PropertyChanged(this, new PropertyChangedEventArgs(nameof(CoreTitleBarPadding)));
             }
         }
+
+        private void TabView_TabDraggedOutside(object sender, Microsoft.Toolkit.Uwp.UI.Controls.TabDraggedOutsideEventArgs e)
```

Comment:
leftover of TabView?

Reply:
yes, sorry, good catch!
TabView worked pretty well except that it doesn't provide a template to customize whole tab header area for specific state (hovered, pressed, etc) and option to display tabs headers at the bottom of page. To have that desired customization it seems it's needed to redefine too much functionality of TabView's high level templates.